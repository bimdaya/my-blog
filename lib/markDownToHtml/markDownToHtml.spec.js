import markDownToHtml from './markDownToHtml';

describe('markDownToHtml()', () => {
  it('Returns string of HTML tags', async () => {
    expect(await markDownToHtml('## Test Assertion')).toMatchSnapshot();
  });
});
