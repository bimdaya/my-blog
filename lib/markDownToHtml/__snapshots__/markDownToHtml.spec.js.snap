// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`markDownToHtml() Returns string of HTML tags 1`] = `
"<h2>Test Assertion</h2>
"
`;
